src/main/java/frc/robot/subsystems/Drive.java

```diff
@@ -86,13 +88,19 @@ public void initDefaultCommand() {
 
     public Command driveNormal() {
         return new Command("driveNormal", this) {
+            private double triggerSpeed = 0;
+            private double thumbstickSpeed = 0;
 
             @Override
             protected void execute() {
-                drive.arcadeDrive(
-                        +Robot.driveController.getTriggerAxis(Hand.kRight)
```

Comment:
If you add `XboxController c = Robot.driveController;` before this, the rest of this function can get easier to read (less verbose)

Reply:
doneeeeeeee